project.clj

```diff
@@ -2,18 +2,16 @@
   :url "https://github.com/status-im/status-react/"
   :license {:name "Eclipse Public License"
             :url  "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.9.0-RC1"]
+  :dependencies [[org.clojure/clojure "1.9.0"]
                  [org.clojure/clojurescript "1.9.946"]
-                 [org.clojure/core.async "0.3.443"]
+                 [org.clojure/core.async "0.4.474"]
                  [reagent "0.7.0" :exclusions [cljsjs/react cljsjs/react-dom cljsjs/react-dom-server cljsjs/create-react-class]]
                  [re-frame "0.10.2"]
-                 [com.andrewmcveigh/cljs-time "0.5.0"]
+                 [com.andrewmcveigh/cljs-time "0.5.2"]
                  [com.taoensso/timbre "4.10.0"]
                  [hickory "0.7.1"]
                  [com.cognitect/transit-cljs "0.8.243"]]
-  :plugins [[lein-cljsbuild "1.1.7"]
-            [lein-figwheel "0.5.14"]
-            [lein-re-frisk "0.5.5"]]
```

Comment:
sometimes i run `lein re-frisk` without running figwheel, can we keep it?

Reply:
I didn't know you could do that. When do you choose the former vs the latter?

Why I removed it: `lein-re-frisk` has a lot of deps, some conflicting with our ones. Also I think it might make startup a little slower.

Reply:
nope, we still have https://github.com/status-im/status-react/pull/3066/files#diff-0fff143854a4f5c0469a3819b978a483R27 `re-frisk-sidecar` it has all deps, and runs re-frisk sever, it makes startup a little slower

Reply:
so deletion of `lein-re-frisk` nothing changed

Reply:
Hmm indeed but it will change after my next bounty as we need to have a dedicated profile for `figwheel` to solve some subtle issues :D 

If you rely on `lein-re-frisk` we can keep it for sure.

Reply:
yes please